medusa/failed_processor.py

```diff
@@ -44,8 +44,8 @@ def process(self):
             parsed = NameParser().parse(releaseName)
         except (InvalidNameException, InvalidShowException):
             self.log(logger.WARNING, u'Not enough information to parse release name into a valid show. '
-                      u'Consider adding scene exceptions or improve naming for: {release}'.format
-                      (release=releaseName))
+                     u'Consider adding scene exceptions or improve naming for: {release}'.format
+                     (release=releaseName))
```

Comment:
`Consider adding scene exceptions (https://github.com/pymedusa/Medusa/wiki/Scene-exceptions-and-numbering)`

Of course the wiki link should be a shorten url